### PR TITLE
Metadata drawer

### LINKED ISSUE #50: metadata drawer
when an arrow icon on a result in the results table is clicked a drawer component should slide out to the right of the table. Drawer will contain meta data for selected raster.

component should be a molecule and will need to include all meta data except the data cited in #49

### PR DESCRIPTION
**Pull request by annehaley**
Resolves #50.

### CODE REVIEW

**Comment by annehaley** [comment]
There's currently one problem for which I'd like advice: the `v-navigation-drawer` is supposed to open and close based on its `value` prop, but I can't get it to close once open with this prop. Instead, I resorted to specifying the show variable with `v-if`, which means we don't get the nice sliding drawer animation. Any tips?

**Review by marySalvi — APPROVED** [review]
Looks good, feel free to merge

File: src/store/index.ts [hunk 1]
```diff
@@ -62,18 +64,37 @@ export const addFootPrint = async (spatialId: number) => {
   footPrints.value.push(res.data);
 };
 
-export const removeFootPrint = (spatialId: number) => {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const removeFootPrint = (_spatialId: number) => {
```
Comment (banesullivan) [inline src/store/index.ts:68]: @annehaley, why this change?

Comment (annehaley) [reply src/store/index.ts:68]: Simply to get rid of the style warnings. I set up each of these functions to be filled in, but since they don't use `spatialId` yet, the linter made these suggestions. This can be undone when the variable is used.
